# DEF-D Fysieke Ontwerpopdracht 2 invul template

## Groep informatie

Aan het einde van deze ontwerpopdracht lever je als groep één gezamenlijk notebook in (als in *PR* op GitHub en de link op Brightspace). Zet jullie groepsnummer, studienummers en namen in onderstaande tabel zodat wij zeker weten dat het groepsvormen goed is gegaan.

| Groep: 27     |  |
| :-------------|:-------------|
| Jinzhou Mo| 6126537 |
| Cedric Peignier| 6577458 |
| Aragon van Doesburg| 6394485 |
| Planning Groep: 27     |Tijdstip / Tijdspanne  |
|---|---|
| Sanity checks 1,2 | 11:30 |
| Sanity check 3 | 12:00 |
| sensor af| 14:00 |
| Calibratie meting| 16:00 |
| af | 17:00 |
| Pauze 1| 11:45-12:00 |
| Pauze 2| 16:00-16:15 |

## *Opdracht 1*: Sanity check 3

$$
V(t) = sin(2\pi f t)
$$
$$
I = C\frac{dV}{dt}
$$
$$
\frac{V_{out}}{R} = C\frac{dV}{dt}
$$
Na differentiatie en herschrijven krijgen we
$$
C = \frac{V_{out}}{V_0 2\pi fR}
$$

In [ ]:
def func(amplitude, frequency, Vout, R):
    return Vout / (amplitude * frequency * R * 2*np.pi)

frequency = 1e5 #Hz
amplitude = 8 #V
R = 1000 #Ohm
Vout = 5 #V

print(func(amplitude, frequency, Vout, R))

9.947183943243459e-10


#### Voeg hier een foto van de schets van je plaatcondensator in. Geef daarbij ook duidelijk de waarde van $A$ en $d$ aan.
<img src="tudelftLogo.png" alt="schets student 1" width="400"/>

In [10]:
# Schrijf hier een python functie die de capaciteit van een plaatcondensator uitrekent. Zorg dat de functie
# als input de ontwerpparameters van de vlakke plaat formule accepteert (oppervlakte, afstand en diëlektrische
# constante) en als output de capaciteit van de vlakke plaat condensator met die ontwerpparameters.
# Je gaat deze formule de rest van de dag gebruiken: controleer hem goed!
import numpy as np
import matplotlib.pyplot as plt
from scipy.optimize import curve_fit

e0 = 8.8541878128e-12

def capacity(er,L,B,d):
    '''
    epsilon = permitiviteit medium
    L = lengte in m
    B = breedte in m
    d = afstand tussen platen in m
    '''
    C = er*e0*B*L/d 
    return C

# Optioneel: schrijf je functie zodanig dat je arrays als input kan geven en (dus) ook arrays als output.

In [15]:
# Gebruik je formule om uit te rekenen wat je verwacht dat de capaciteit van jullie vlakke plaat condensatoren
# gaat zijn.

L = 0.3 # m
B = 0.3 # m 
d = 0.001 # m
er = 1.78 
C = capacity(er,L,B,d)

print(f"Berekende capaciteit plaatcondensator groep 11 is {C} F")



Berekende capaciteit plaatcondensator groep 11 is 1.41844088761056e-09 F


#### Voeg hier een foto van je plaatcondensator en van de meting op de scope in. Dus 2 foto's.
![scope meting 1](tudelftLogo.png)
![scope meting 2](tudelftLogo.png)

### Meting capaciteit
Vul onderstaande tabel in. Bij Sanity check 2 heb je afgeleid hoe je van berekende capaciteit naar berekend voltage gaat en terug. 


| Namen         | Berekende capaciteit (F)  | Berekend Voltage | Gemeten Voltage | Gemeten capaciteit   |
|-------------|-------------:|-------------:|-------------:|-------------:|
| groep 27| $1.42\cdot 10^{-9}$ |? | ? |? |

## *Opdracht 2:* Synthese.

#### Voeg hier voor een foto van je schets(en) van minimaal drie sensoren in.
![drie schetsen](tudelftLogo.png)


## Opdracht 3: Welke sensor gaan jullie maken?

Pas deze tekst aan: Wij gaan een sensor maken die X meet.

## Opdracht 4: Uitgewerkte schets (afmetingen!) en plot van de ontwerpgrafiek.

#### Voeg hier een foto van jullie uitgewerkte  schets van de sensor die jullie gaan maken in
![uitgewerkte schets](tudelftLogo.png)

In [3]:
# Plot hier de verwachte ontwerpgrafiek van jullie sensoren.

## Opdracht 5: Invullen tabel LTSpice.

|         | Capaciteit waarde (pF)  | LTSpice berekend output Vpp (mV)|
|-------------|-------------:|-------------:|
| minmale capaciteit| ? |? |
| gemiddelde capaciteit| ? |? |
| maximale capaciteit| ? |? |

## Opdracht 6: Foto's werkende opstelling.

#### Voeg hier de foto's van je sensoren in actie in.
![meting 1](tudelftLogo.png)
![meting 2](tudelftLogo.png)

## Opdracht 7: Plot de kalibratiegrafiek.

In [4]:
# Plot hier je gemeten kalibratiegrafiek en plot in dezelfde figuur ook je verwachte ontwerpgrafiek.

## Opdracht 8: Vul de tekst aan.

Onze sensor kan nog verbetered worden door de volgende verbeteringen (minimaal 2):

* verbetering 1
* verbetering 2

## Opdracht 9: Uiteindelijke ontwerp en kalibratiegrafiek.

#### Voeg hier een foto van je uiteindelijke sensor in.
![foto sensor](tudelftLogo.png)

In [5]:
# Plot hier je oorspronkelijke en je uiteindelijke kalibratiegrafieken, fit daar een bijpassende lijn door
# en zet ook de verwachte ontwerpgrafiek erbij

## Opdracht 10: Vul hieronder de conclusie in.

Vul hier de conclusie in

## Opdracht 11: Kopieer hieronder de leerdoelen.
1. Het kunnen gebruiken van de eigenschappen van een elektrische component in een ontwerpproces om een fysische grootheid te meten.
2. Kennis hebben van gedrag van elektrische componenten en meetopstellingen.
3. Onderdelen van een datasheet zoals een kalibratiegrafiek opstellen, gevoeligheid en onzekerheid beschrijven.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 1 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 2 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 3 beheersen.